### ISSUE TITLE
Header Image

### ISSUE THREAD

**Issue by mp1pro** [shiina-ringo/dasmoto#1]
The header image was a bit larger that the design in the mock-up. It can be decreased by reducing the font-size in banner to 50px like so

`.banner {font-size:50px;}`

**Comment by shiina-ringo** [comment]
I agree it doesn't look like the mock-up, but the spec (https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/dasmotos-arts_redline.jpg) required the font size to be 100px. Perhaps this was a typo in the spec document?

**Comment by shiina-ringo** [comment]
Ah, never mind. It seems like setting the font-size for the h1 element directly versus the entire banner class produces different results. I refactored the code to do the former, and now the final result looks just like the mock-up. Thanks for the feedback!
